libs/ebpfnetsh/elf.cpp

```diff
@@ -325,8 +326,24 @@ handle_ebpf_show_verification(
         }
     }
 
+    ebpf_verification_verbosity_t verbosity;
+    switch (level) {
+    case VL_NORMAL:
+        verbosity = EBPF_VERIFICATION_VERBOSITY_NORMAL;
+        break;
+    case VL_INFORMATIONAL:
+        verbosity = EBPF_VERIFICATION_VERBOSITY_INFORMATIONAL;
+        break;
+    case VL_VERBOSE:
+        verbosity = EBPF_VERIFICATION_VERBOSITY_VERBOSE;
+        break;
+    default:
+        std::cerr << "Invalid verbosity level" << std::endl;
+        return ERROR_SUPPRESS_OUTPUT;
```

Comment:
This can't happen without a code bug, suggest just asserting or something, since otherwise this is an untestable code path that lowers code coverage numbers, and at minimum adds a string which is never used, to the size of the binary.